packages/next/client/image.tsx

```diff
@@ -223,13 +231,6 @@ export default function Image({
         ).join(',')}.`
       )
     }
-    if (!VALID_LOADING_VALUES.includes(loading)) {
-      throw new Error(
-        `Image with src "${src}" has invalid "loading" property. Provided "${loading}" should be one of ${VALID_LOADING_VALUES.map(
-          String
-        ).join(',')}.`
-      )
-    }
```

Comment:
Why was the `loading=eager` validation removed?

Reply:
Whoops, fixed.